README.md

```diff
@@ -5,3 +5,50 @@
 [![OpenIssue](https://img.shields.io/github/issues/CeresDB/sqlness)](https://github.com/CeresDB/sqlness/issues)
 
 **SQL** integration test har**NESS**
+
+An ergonomic, opinionated framework for SQL integration test.
```

Comment:
What does "opinionated" stand for here

Reply:
https://stackoverflow.com/questions/802050/what-is-opinionated-software

Reply:
TIL 👍